app/utilities/schema.py

```diff
@@ -42,7 +42,7 @@ def load_schema_file(schema_file, language_code=None):
 
     try:
         with open(schema_path, encoding='utf8') as json_data:
-            return json.load(json_data)
+            return json.load(json_data, use_decimal=True)
```

Comment:
Have you done any testing with decimals saved using a master version of runner, and then resuming with this change?

Reply:
Yes this morning, behaves as expected 👍